docs/source/advanced/profiler.rst

```diff
@@ -212,6 +212,18 @@ XLA Profiling
 :class:`~pytorch_lightning.profiler.xla.XLAProfiler` will help you debug and optimize training
 workload performance for your models using Cloud TPU performance tools.
 
+.. code-block:: python
+
+    # by passing the `XLAProfiler` alias
+    trainer = Trainer(..., profiler="xla")
+
+    # or by passing an instance
+    from pytorch_lightning.profiler.xla import XLAProfiler
```

Comment:
should we make the profiler imports all be exposed at the lightning.profiler module? this would be consistent with what we do for other components. fyi @daniellepintz

Reply:
Yup, updated! Thanks